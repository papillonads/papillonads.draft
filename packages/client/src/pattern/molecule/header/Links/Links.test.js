import defaultRender from './Links.render'

describe('<Links />', () => {
  describe('Snaphot', () => {
    test('must match defaultRender', () => {
      expect(global.renderToJSON(defaultRender)).toMatchSnapshot()
    })
  })
})
