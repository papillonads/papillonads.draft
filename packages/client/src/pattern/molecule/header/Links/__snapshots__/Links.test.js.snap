// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Links /> Snaphot must match defaultRender 1`] = `null`;
